package.json

```diff
@@ -45,6 +45,7 @@
     "chroma-js": "^2.1.2",
     "color": "^3.1.0",
     "react-indiana-drag-scroll": "^1.7.0",
+    "react-native-device-info": "^10.3.0",
```

Comment:
Tried to find a way to detect dynamic islands without installing any additional dependencies, but didn't manage to find a great solution. 

That being said, I've used react-native-device-info before and  I'm a fan.